templates/zerver/app/navbar_alerts.html

```diff
@@ -1,33 +1,33 @@
 <div id="panels">
     <div data-process="notifications" class="alert alert-info">
-        <span class="close" data-dismiss="alert" aria-label="{{ _('Close') }}">&times;</span>
         <div data-step="1">
             {% trans %}Zulip needs your permission to
-            <a class="request-desktop-notifications alert-link">enable desktop notifications.</a>
+            <a class="request-desktop-notifications alert-link" role="button" tabindex=0>enable desktop notifications.</a>
             {% endtrans %}
+            <span class="close" data-dismiss="alert" aria-label="{{ _('Close') }}" role="button" tabindex=0>&times;</span>
         </div>
         <div data-step="2" style="display: none">
             {{ _('We strongly recommend enabling desktop notifications. They help Zulip keep your team connected.') }}
             <span class="buttons">
-                <a class="alert-link request-desktop-notifications">{{ _('Enable notifications') }}</a>
+                <a class="alert-link request-desktop-notifications" role="button" tabindex=0>{{ _('Enable notifications') }}</a>
                 &bull;
-                <a class="alert-link exit">{{ _('Ask me later') }}</a>
+                <a class="alert-link exit" role="button" tabindex=0>{{ _('Ask me later') }}</a>
                 &bull;
-                <a class="alert-link reject-notifications">{{ _('Never ask on this computer') }}</a>
+                <a class="alert-link reject-notifications" role="button" tabindex=0>{{ _('Never ask on this computer') }}</a>
             </span>
+            <span class="close" data-dismiss="alert" aria-label="{{ _('Close') }}" role="button" tabindex=0>&times;</span>
```

Comment:
You didn't need to duplicate this line; it just belongs outside the `data-step` divs

Reply:
@timabbott, I think if we have to avoid duplication we need to find other way because otherwise it produces this result.
![Screenshot from 2020-08-15 05-17-10](https://user-images.githubusercontent.com/30050220/90300276-4bc7fb00-deb7-11ea-829d-4904bdeaa2cc.png)

Also, it mess with the tab order.
![Peek 2020-08-15 05-18](https://user-images.githubusercontent.com/30050220/90300457-24256280-deb8-11ea-92f8-d24aa51ac213.gif)